src/Dom/Dom.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace HeadlessChromium\Dom;
+
+use HeadlessChromium\Communication\Message;
+use HeadlessChromium\Page;
+
+class Dom extends Node
+{
+    /**
+     * Dom constructor.
```

Comment:
Please don't include comments like this that don't add any extra information. I know things like this may be present on existing code, but I'd like to make sure new code is of a better quality. :)